backup.c

```diff
@@ -185,14 +185,18 @@ do_backup_database(parray *backup_list, pgBackupOption bkupopt)
 	/*
 	 * To take incremental backup, the file list of the latest validated
 	 * full database backup is needed.
+	 * There a problem of increment backup.
+	 * IF a physical backup file is deleted(rm command, etc.) other than the pg_rman delete command of pg_rman, 
```

Comment:
IF is capital. Is it intentional?

Reply:
It'll happen even if using pg_rman delete command. The delete command has the dangerous '--force' option.

Reply:
Thank you for you comment. I already fix it..

Reply:
Thanks.